app_legacy/Helpers/render/user.php

```diff
@@ -147,23 +147,52 @@ function renderUserCard(string|array $user): string
     return $tooltip;
 }
 
+function getCompletedAndIncompletedSetsCounts(array $userCompletedGamesList): array
+{
+    $completedSetsCount = 0;
+    $incompletedSetsCount = 0;
+
+    $numItems = count($userCompletedGamesList);
+    for ($i = 0; $i < $numItems; $i++) {
+        $nextMaxPossible = $userCompletedGamesList[$i]['MaxPossible'];
+        $nextNumAwarded = $userCompletedGamesList[$i]['NumAwarded'];
+
+        // Don't divide by 0.
+        if ($nextNumAwarded == 0 || $nextMaxPossible == 0) {
+            continue;
+        }
+
+        $pctAwardedNormal = ($nextNumAwarded / $nextMaxPossible) * 100.0;
+        if ($pctAwardedNormal == 100) {
```

Comment:
`if ($nextNumAwarded == $nextMaxPossible)`?

Would avoid doing the multiplication and division.

Reply:
Resolved in https://github.com/RetroAchievements/RAWeb/pull/1492/commits/5353618147739ece716f4f3e6ef5991ddf1d9098.